.github/workflows/iroha2-dev-pr.yml

```diff
@@ -31,9 +31,14 @@ jobs:
         run: mold -run cargo test --quiet --features mock -- --ignored --skip unstable_network --test-threads 2
         working-directory: core/test_network
       - name: Run iroha_actor deadlock detection tests
-        working-directory: actor
         run: mold -run cargo test --quiet --features deadlock_detection -- --skip unstable_network --test-threads 2
+        working-directory: actor
         if: always()
+      - name: Verify wasm smartcontracts are able to build
```

Comment:
Just a check that there is no regression, i.e. that smartcontracts can be built.
Should this check be placed somewhere else?